scripts/lib/CIME/utils.py

```diff
@@ -26,7 +26,6 @@ def expect(condition, error_msg, exc_type=SystemExit, error_prefix="ERROR:"):
         if logger.isEnabledFor(logging.DEBUG):
             import pdb
             pdb.set_trace()
-        logger.error("%s %s", error_prefix,error_msg, exc_info=True)
```

Comment:
Is this removal intentional? I assume so, but just wanted to double-check since it seems unrelated to the rest of the PR